src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem';
+
+interface Props {
+  todos: Todo[];
+}
+
+export const TodoList: React.FC<Props> = ({ todos }) => (
+  <section className="todoapp__main">
```

Comment:
Add a condition to hide the list and footer when there are no todos.

Reply:
> Add a condition to hide the list and footer when there are no todos.

I did this in App component